tests/api/test_addon_uploads.py

```diff
@@ -95,3 +95,150 @@ def test_upload_unlisted_extension(base_url, selenium, session_auth):
     # get the edit url for the add-on to verify that it was created and visible in devhub
     r = requests.get(resp['edit_url'], cookies={"sessionid": session_auth}, timeout=10)
     assert r.status_code == 200
+
+
+@pytest.mark.serial
+@pytest.mark.create_session('api_user')
+def test_submit_extension_with_invalid_uuid_format(base_url, session_auth):
+    """The UUID format doesn't match the expected format for the uuid value"""
+    uuid = [
+        'some-invalid-uuid',
+        {'upload': 'd4ce752a971b4a5aafcd175122726431'},
+        None,
+        '',
+    ]
+    for item in uuid:
+        data = {'version': {'upload': item}}
+        create_addon = requests.post(
+            url=f'{base_url}{_addon_create}',
+            headers={
+                'Authorization': f'Session {session_auth}',
+                'Content-Type': 'application/json',
+            },
+            data=json.dumps(data),
+        )
+        # capture the response details to ease debugging
+        print(
+            f'For UUID "{item}": Response status is '
+            f'{create_addon.status_code}; {create_addon.text}\n'
+        )
```

Comment:
I don't think this will print if the test is true. Is that how you want it to work?

Reply:
Mm, this should be printed regardless of the test passing or failing - see in the [html report](https://output.circle-artifacts.com/output/job/a071b6ec-3a50-40bd-ac70-e1d576dce4f2/artifacts/0/api_submission_tests.html)